x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -300,30 +302,6 @@ export const useCasesColumns = ({
         return getEmptyTagValue();
       },
     },
-    ...(isSelectorView
```

Comment:
This section was merely moved down to allow the severity field to show before the status.